src/core/function.js

```diff
@@ -399,15 +399,17 @@ var PDFFunction = (function PDFFunctionClosure() {
       var range = IR[2];
       var code = IR[3];
 
-      var compiled = (new PostScriptCompiler()).compile(code, domain, range);
-      if (compiled) {
+      if (!!PDFJS.isEvalSupported) {
```

Comment:
I'm assuming that we may also want to do something similar to https://github.com/mozilla/pdf.js/blob/23a4174163cba5709f05b04b371ee67b1d0148ca/src/display/font_loader.js#L401 and https://github.com/mozilla/pdf.js/blob/23a4174163cba5709f05b04b371ee67b1d0148ca/src/display/font_loader.js#L332 here as well?